third_party/move/move-prover/boogie-backend/src/options.rs

```diff
@@ -140,6 +140,11 @@ pub struct BoogieOptions {
     pub random_seed: usize,
     /// The number of cores to use for parallel processing of verification conditions.
     pub proc_cores: usize,
+    /// The number of shards to split the verification problem into.
+    pub shards: usize,
+    /// If there are shards, specifies to only run the given shard. Shards are numbered
+    /// starting at 1.
```

Comment:
Starting at 1 is for readability of shard info?

Reply:
Yeah, generally numbering starts at 1